core/src/test/scala/integration/kafka/api/SaslSetup.scala

```diff
@@ -43,32 +43,33 @@ trait SaslSetup {
   private val workDir = TestUtils.tempDir()
   private val kdcConf = MiniKdc.createConfig
   private var kdc: MiniKdc = null
+  var serverKeytabFile: Option[File] = None
+  var clientKeytabFile: Option[File] = None
```

Comment:
@ijuma Thank you for the review. Don't think we need an accessor since subclasses don't use them, so I have made the vars private. I have done 2).